2996_reflection/reflection.md

```diff
@@ -3008,6 +3008,63 @@ Change the grammar for `$operator-or-punctuator$` in paragraph 1 of [lex.operato
 ```
 :::
 
+### [basic.pre]{.sref} Preamble {-}
+
+Add type aliases and namespace aliases to the list of entities in paragraph 3. As drive-by fixes, remove "variable", "object", "reference", and "template specialization"; replace "class member" with "non-static data member", since all other cases are subsumed by existing one. Add "template parameters" and "`$init-capture$`s", which collectively subsume "packs".
+
+::: std
+[3]{.pnum} An _entity_ is a [value, object, reference]{.rm} [variable,]{.addu} structured binding, function, enumerator, type, [type alias]{.addu}, [class]{.rm} [non-static data]{.addu} member, bit-field, template, template specialization, namespace, [namespace alias, template parameter]{.addu}, or [`$init-capture$`]{.addu} [pack]{.rm}.
```

Comment:
I feel like we should have a note for other things that are entities as a follow on from here. Like a function parameter pack is an entity because it is a pack of variables or something?

Reply:
I believe that Davis's Grand Design wants us to move towards "entity" being roughly "eternal thing whose declarations are found by name lookup" - a pack can be found by name lookup, ergo entity.